splash/__init__.py

```diff
@@ -79,20 +84,28 @@ def create_app(db=None):
     from splash.auth.oauth_resources import OAuthResource
     api.add_resource(OAuthResource, "/api/tokensignin", resource_class_kwargs={'user_service': app.user_service})
 
+    @app.route('/metrics/')
+    def metrics():
+        return Response(prometheus_client.generate_latest(),
+                        mimetype=CONTENT_TYPE_LATEST)
+
```

Comment:
Yes this is where the endpoint is added.

Reply:
Missed that. Great.